src/crystal/system/thread_linked_list.cr

```diff
@@ -1,13 +1,13 @@
-require "thread/mutex"
+require "./thread_mutex"
 
 # :nodoc:
-class Thread
+class Crystal::Thread
```

Comment:
Could we revert this change and leave `Thread` instead of `Crystal::Thread`? I don't think we can merge this PR with this. But @waj and @bcardiff might be able to provide more context about why we don't want to make this change.

Reply:
It should at least be another PR, honestly.

I like the change though.

Reply:
Maybe. I just don't think we should delay Windows support just because some want to rename a class and some don't.

Reply:
Okay, I'll revert to `Thread`.